# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dosfstools/compile.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./dosfsck/boot.c.orig	2005-03-11 22:26:53.000000000 -0300
+++ ./dosfsck/boot.c	2005-03-11 22:31:02.000000000 -0300
@@ -13,7 +13,7 @@
 #include "dosfsck.h"
 #include "io.h"
 #include "boot.h"
-
+#define MSDOS_FAT12 4084 /* maximum number of clusters in a 12 bit FAT */
 
 #define ROUND_TO_MULTIPLE(n,m) ((n) && (m) ? (n)+(m)-1-((n)-1)%(m) : 0)
     /* don't divide by zero */
